Remotion/Data/DomainObjects/Persistence/Rdbms/SqlServer/Model/Building/SqlStorageTypeInformationProvider.cs

```diff
@@ -226,6 +228,18 @@ public IStorageTypeInformation GetStorageType (object? value)
       return null;
     }
 
+    private StorageTypeInformation GetDateTimeStorageTypeInformation (Type dotNetType, bool isNullableInDatabase)
```

Comment:
this PR might benefit from getting done first, because we then have the correct place to add the attributes.

Reply:
On the other hand, we don't have test data here. I think I can live with that. But if you just take this API-update and already do it in the other PR, we probably are cleaner since we get the tests first and the default-api second.